pkg/kubernetes/scripts/projects.js

```diff
@@ -940,10 +977,13 @@
 
                 return methods.check(request, { "metadata.name": "#project-new-name" })
                     .then(function() {
-                        return methods.create(request)
-                            .then(function() {
-                                return projectData.shareImages(name, fields.access === "shared");
-                            });
+                        return methods.create(request);
+                    })
+                    .then(function(){
+                        return projectData.shareImages(name, fields.access === "shared");
+                    })
+                    .then(function(){
+                        return projectData.makeRegistryAdmin(name, dialogData.currUser);
```

Comment:
@stefwalter @aweiteka
for time being we need this to make registry-admin useful in our UI

@aweiteka IMO we need to add a project template which will not create a 'admin' policy. ok  ?